XenModel/Utils/Helpers.cs

```diff
@@ -488,7 +487,7 @@ public static bool PostStockholm(Host host)
 
         public static bool PostStockholm(string platformVersion)
         {
-            return platformVersion != null && productVersionCompare(platformVersion, "3.2.50") >= 0;
+            return platformVersion != null && productVersionCompare(platformVersion, "3.2.1") >= 0;

```

Comment:
This makes it the same as YangtzeOrGreater, which gives incorrect results. I think we probably need to keep it as it was and rename the overloads of PostStockholm to Post82x instead of adding a new method PostYangzte.

Reply:
done and rebased on master